packages/svelte/scripts/generate-dts.js

```diff
@@ -1,14 +1,29 @@
+import * as fs from 'fs';
 import { createBundle } from 'dts-buddy';
 
+fs.readdirSync('src/runtime', { withFileTypes: true })
+	.filter((dirent) => dirent.isDirectory())
+	.forEach((dirent) => fs.writeFileSync(`${dirent.name}.d.ts`, `import 'types/index.d.ts';`));
```

Comment:
Can we replace this with an explicit list (which `modules` below could be derived from) so that we don't end up creating garbage files if we add a new directory for whatever reason?